entrypoint.sh

```diff
@@ -135,7 +135,8 @@ install_node()
     fi
 
     wget -q -O nvm-install.sh https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh
-    if [ ! "fabc489b39a5e9c999c7cab4d281cdbbcbad10ec2f8b9a7f7144ad701b6bfdc7  nvm-install.sh" = "$(sha256sum nvm-install.sh)" ]; then
+    sha256sum -c checksum --status --strict --ignore-missing
```

Comment:
if this returns non-zero (i.e mismatched checksum), the execution will stop then because the script is run with `set -e` and the user won't see the error message from below.

I believe you should be able to do `if ! sha256sum .....; then ...` instead to have it working.

I am also not a fan of `--ignore-missing`, as it could be error prone. If, for instance, the filename changes, the check may still pass silently, leaving the user unprotected. We could have two separate checksum files to avoid having to use `--ignore-missing`

Reply:
Does this the new syntax (with the grep) looks good ?

Reply:
yes, thanks!